File-Formats/DocIO/Word-document/Compare Word documents.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Compare Word documents in C# | DocIO | Syncfusion
+description: Learn how to compare two Word documents in C# using .NET Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Compare Word documents in C#
+
+Comparing two documents allows you to identify differences and similarities between them. The .NET Word library (DocIO) allows you to compare two Word documents and highlight the following changes as tracked changes
+*	Insertions
+*	Deletions
+*	Formatting
+
+N> Comparing two Word documents supported in DOCX format only.
```

Comment:
Add this as 1st  Note

DocIO performs word level comparison while comparing two Word documents.

Reply:
updated